src/translations/main_nl.ts

```diff
@@ -0,0 +1,946 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+<context>
+    <name>AutoUpdater</name>
+    <message>
+        <source>No ssl configuration!
+OpenSSL might not be installed.</source>
+        <translation>Geen SSL-configuratie aangetroffen!
+Mogelijk is OpenSSL niet geïnstalleerd.</translation>
+    </message>
+    <message>
+        <source>Failed to save the Birdtray installer:
+</source>
+        <translation>Het opslaan van de Birdtray-installatiewizard is mislukt:
+</translation>
+    </message>
+    <message>
+        <source>Failed to download the Birdtray installer:
+</source>
+        <translation>Het downloaden van de Birdtray-installatiewizard is mislukt:
+</translation>
+    </message>
+    <message>
+        <source>Installer download failed</source>
+        <translation>Downloaden mislukt</translation>
+    </message>
+    <message>
+        <source>Invalid redirect: </source>
+        <translation>Ongeldige doorverwijzing: </translation>
+    </message>
+    <message>
+        <source>Update failed</source>
+        <translation>Update mislukt</translation>
+    </message>
+    <message>
+        <source>Failed to start the Birdtray installer.</source>
+        <translation>Het starten van de Birdtray-installatiewizard is mislukt.</translation>
+    </message>
+</context>
+<context>
+    <name>BirdtrayApp</name>
+    <message>
+        <source>A free system tray notification for new mail for Thunderbird</source>
+        <translation>Een gratis systeemvakprogramma dat meldingen toont van Thunderbird</translation>
+    </message>
+    <message>
+        <source>Display the contents of the given mork database.</source>
+        <translation>Bekijk de inhoud van de opgegeven mork-databank.</translation>
+    </message>
+    <message>
+        <source>databaseFile</source>
+        <translation>databankbestand</translation>
+    </message>
+    <message>
+        <source>Decode an IMAP Utf7 string.</source>
+        <translation>Decodeer een IMAP-Utf6-string.</translation>
```

Comment:
```suggestion
        <translation>Decodeer een IMAP-Utf7-string.</translation>
```
This doesn't look right.

Reply:
Huh? I'm pretty sure I did not type a '6' there, but will fix that tomorrow then.